### PR TITLE
add HTTP 214: Transformation Applied

### PR DESCRIPTION
**Pull request by karlhiramoto**
Add HTTP 214
* https://datatracker.ietf.org/doc/html/rfc7234#section-5.5.6
* https://webconcepts.info/concepts/http-warn-code/214

### CODE REVIEW

**Comment by karlhiramoto** [comment]
If you don't like the image in PR,  Alternative 1:
image

**Comment by rogeriopvl** [comment]
Hi @karlhiramoto 

Thanks for the contribution! I do actually prefer this last image. Can you update the PR to use this one?

**Comment by karlhiramoto** [comment]
@rogeriopvl  image updated.

File: public/images-original/214.jpg
Comment (rogeriopvl) [inline public/images-original/214.jpg:1]: @karlhiramoto the image is not centered, can you adjust it please?

Thanks

**Review by rogeriopvl — CHANGES_REQUESTED** [review]

File: public/images-original/214.jpg
Comment (karlhiramoto) [reply public/images-original/214.jpg:1]: Done

**Comment by rogeriopvl** [comment]
@karlhiramoto thanks, merging now. I'll improve the description content in a separate commit.

**Comment by ddelange** [comment]
this is awesome
[https://http.cat/202](https://http.cat/202)

**Pull request merged**